src/internal/m365/service/groups/backup.go

```diff
@@ -34,18 +34,26 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api/graph"
 )
 
-func ProduceBackupCollections(
+type groupsBackup struct{}
+
+// NewBackup provides a struct that matches standard apis
+// across m365/service handlers.
+func NewBackup() *groupsBackup {
+	return &groupsBackup{}
+}
+
+func (groupsBackup) ProduceBackupCollections(
 	ctx context.Context,
 	bpc inject.BackupProducerConfig,
 	ac api.Client,
 	creds account.M365Config,
 	su support.StatusUpdater,
 	counter *count.Bus,
 	errs *fault.Bus,
-) ([]data.BackupCollection, *prefixmatcher.StringSetMatcher, error) {
+) ([]data.BackupCollection, *prefixmatcher.StringSetMatcher, bool, error) {
```

Comment:
We should put the original comment here

```
// canUsePreviousBacukp can be always returned true for groups as we
// return a tombstone collection in case the metadata read fails
```

Reply:
added that to the interface